README.md

```diff
@@ -295,7 +297,7 @@ The content viewset usually doesn't require any additional routes, so you can le
 
 ## Remote
 
-Remotes provide metadata about how content should be downloaded into Pulp, such as the URL of the remote source, the download policy, and some authentication settings. The base ``Remote`` class provided by Pulp Platform provides support for concurrent downloading of remote content. 
+Remotes provide metadata about how content should be downloaded into Pulp, such as the URL of the remote source, the download policy, and some authentication settings. The base ``Remote`` class provided by Pulp Platform provides support for concurrent downloading of remote content.
```

Comment:
Can this be line wrapped please? Currently this is one really wide line which doesn't conform with the docs line width max of 100 chars.